client/src/components/Requests/NextRequest/NextRequest.tsx

```diff
@@ -0,0 +1,43 @@
+import { Paper, Typography, Grid, IconButton } from '@material-ui/core';
+import Box from '@material-ui/core/Box';
+import Request from '../../../interface/Request';
+import useStyles from './useStyles';
+import Avatar from '@material-ui/core/Avatar';
+import SettingsIcon from '@material-ui/icons/Settings';
+import RequestListing from '../RequestListing/RequestListing';
+import RequestMenu from '../RequestMenu/RequestMenu';
+
+interface Props {
+  request: Request | null | undefined;
+}
+
+const NextRequest = ({ request }: Props): JSX.Element => {
+  const classes = useStyles();
+
+  if (request) {
+    return (
+      <Grid className={classes.root} component={Paper} elevation={2}>
+        <Box>
+          <Box display="flex" justifyContent="space-between">
+            <Typography className={classes.heading}>your next booking:</Typography>
+            <RequestMenu request={request} />
+          </Box>
+          <Grid className={classes.listing}>
+            <RequestListing request={request} component={'nextBooking'}></RequestListing>
+          </Grid>
+        </Box>
+      </Grid>
+    );
+  }
+  return (
+    <Grid className={classes.root} component={Paper} elevation={2}>
+      <Box>
```

Comment:
If we don't need these additional Box components, maybe we can remove them? :)

Reply:
Gone.